src/ONYXKEYS.ts

```diff
@@ -402,6 +402,8 @@ const ONYXKEYS = {
         EXIT_SURVEY_REASON_FORM_DRAFT: 'exitSurveyReasonFormDraft',
         EXIT_SURVEY_RESPONSE_FORM: 'exitSurveyResponseForm',
         EXIT_SURVEY_RESPONSE_FORM_DRAFT: 'exitSurveyResponseFormDraft',
+        WORKSPACE_NEW_TAX_FORM: 'workspaceNewTaxForm',
+        WORKSPACE_NEW_TAX_FORM_DRAFT: 'workspaceNewTaxFormDraft',
```

Comment:
Yes, it might not be needed.

It [was decided](https://docs.google.com/document/d/12FZycslrIE95QoKM8DZ37H_wLbH9w1jt3uA-a6BCqtc/edit?disco=AAABI7i-rdI) that we don't want to store drafts anyway.

Reply:
I think we need to stay with the current approach. Of course, I can rename it if needed.

We can't simply extract it for two reasons:
- validation and submission while editing works on a per-value basis, while creating a new rate works for all values together (Value & Name),
- we need deep links to the Value and Name screens while editing.